package.json

```diff
@@ -43,15 +40,14 @@
   },
   "homepage": "https://github.com/libp2p/js-libp2p-webrtc-direct#readme",
   "engines": {
-    "node": ">=12.0.0",
-    "npm": ">=6.0.0"
+    "node": ">=14.0.0"
   },
   "devDependencies": {
     "aegir": "^30.3.0",
```

Comment:
updating to aegir 31+ causes a segmentation fault running the tests. I think this is related to `wrtc`, but I will create an issue and investigate this separately to avoid blocking the dependencies updates in the bundle

Reply:
Have you tried updating the test command when upgrading aegir?

```
npx nyc -s npm run test:node -- --bail
```

to

```
npx ageir test -t node --cov --bail
```

I saw something similar in https://github.com/ipld/js-ipld-ethereum/pull/71#issuecomment-815979718 and this fixed it.

Reply:
Yes, this does also happens to me locally not running `nyc`. Anyway, just tried locally with a single test case:

```
➜  js-libp2p-webrtc-direct git:(chore/update-deps-12-04-2021) ✗ npx aegir test -t node
Test Node.js
listener started on: /ip4/127.0.0.1/tcp/12345/http/p2p-webrtc-direct


  dial
    ✓ dial on IPv4 (6109ms)


  1 passing (6s)

Command was killed with SIGSEGV (Segmentation fault): mocha test/node.{js,ts} test/**/*.spec.{js,ts} --ui bdd --timeout=5000
```